includes/admin/home/notices/class-sensei-home-notices.php

```diff
@@ -240,10 +240,7 @@ private function get_deactivated_plugin_update_notice( array $plugin_data ) {
 
 		$notice['actions'][] = [
 			'label' => __( 'Activate', 'sensei-lms' ),
-			'url'   => wp_nonce_url(
-				self_admin_url( 'plugin.php?action=activate&plugin=' . $plugin_file ),
-				'activate-plugin_' . $plugin_file
-			),
+			'url'   => add_query_arg( '_wpnonce', wp_create_nonce( 'activate-plugin_' . $plugin_file ), self_admin_url( 'plugins.php?action=activate&plugin=' . $plugin_file ) ),
```

Comment:
We'll need this for the update link as well, if not decoding later.

Reply:
Good catch! Fixed in [dbcc304](https://github.com/Automattic/sensei/pull/5991/commits/dbcc304225e2dbc608e1e920c535eaab94e20c67)